scripts/item-descriptions.lua

```diff
@@ -0,0 +1,75 @@
+-- Holds custom descriptions for view-item-info
+-- By PeridexisErrant
+
+print("This is a content library; calling it does nothing.")
```

Comment:
This will be run when `dfhack.script_environment` is called as well, which probably isn't what you intended. Since this file isn't intended to be used as a script at all, it should probably go in `hack/lua` (`library/lua` in the source tree) or a subdirectory of it (`items`?), which can then be used with `require()`.